ra/ra.go

```diff
@@ -738,20 +734,32 @@ func (ra *RegistrationAuthorityImpl) checkCertificatesPerFQDNSetLimit(ctx contex
 	return nil
 }
 
+func (ra *RegistrationAuthorityImpl) checkTotalCertificatesLimit() error {
+	totalCertLimits := ra.rlPolicies.TotalCertificates()
+	ra.tiMu.RLock()
+	defer ra.tiMu.RUnlock()
+	// If last update of the total issued count was more than a minute ago,
+	// or not yet updated, fail.
+	if ra.clk.Now().After(ra.totalIssuedLastUpdate.Add(1*time.Minute)) ||
```

Comment:
Making the maximum age the same duration as the time in between updates seems like it'll cause false positives, i.e. when a new update has just started but has no yet written to `ra.totalIssuedCount` or is waiting to acquire the lock that `checkTotalCertificatesLimit` is holding.

Was there a specific reason for making these exactly the same?